frontend/src/scenes/pipeline/destinationsLogic.tsx

```diff
@@ -157,11 +153,26 @@ export const pipelineDestinationsLogic = kea<pipelineDestinationsLogicType>([
                 return
             }
             if (destination.backend === 'plugin') {
-                actions.toggleEnabledWebhook({ destination: destination, enabled: enabled })
+                actions.toggleNodeWebhook({ destination: destination, enabled: enabled })
             } else {
-                actions.toggleEnabledBatchExport({ destination: destination, enabled: enabled })
+                actions.toggleNodeBatchExport({ destination: destination, enabled: enabled })
             }
         },
+        deleteNode: async ({ destination }) => {
+            await deleteWithUndo({
```

Comment:
we can't have undo here for batch exports if I understood @tomasfarias  correctly - we've already deleted the schedule and undo won't re-create it

Reply:
Yeah, just realized this, these endpoints are shaped differently. DB-wise both are soft-delete, so with some endpoint reshaping (involving `create_schedule()`) we could support the "I accidentally deleted my export, can I get it back" use case with an "Undo" button – but for now will restore your logic that I removed